### DISCUSSION TITLE
Tailwindcss logical

### DISCUSSION THREAD

**Discussion by vltansky** [dcastil/tailwind-merge#62] [Help]
Hey, could you take a look / review https://github.com/vltansky/tailwind-merge-logical-plugin.
Plugin for https://github.com/stevecochrane/tailwindcss-logical.

**Comment by dcastil** [comment]
Just checked it out and looks good, at least from the tailwind-merge perspective. 😊

Two small thoughts:
- I released tailwind-merge v0.9.0, it would make sense to broaden the peer dependency requirements to `"tailwind-merge": "~0.8.0 || ~0.9.0"`. There are no breaking changes in v0.9.0.
- I introduced theming in v0.9.0, so you can use `fromTheme('spacing')` or `fromTheme('borderRadius')` where appropriate. This way devs can modify these scales and tailwind-merge-logical-plugin will pick these up automatically. You can read more about `fromTheme` [here](https://github.com/dcastil/tailwind-merge/tree/v0.9.0#fromtheme).
